index.html

```diff
@@ -6,6 +6,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
+    <link rel="stylesheet" href="css/flexboxgrid.min.css" type="text/css">
```

Comment:
I don't See the flexboxgrid.min.css but, pls remove this link, we are gone make own flexbox designs